src/main/kotlin/com/github/devcordde/devcordbot/database/_Entities.kt

```diff
@@ -23,6 +23,7 @@ import org.jetbrains.exposed.dao.Entity
 import org.jetbrains.exposed.dao.EntityClass
 import org.jetbrains.exposed.dao.id.EntityID
 import java.time.Instant
+import java.util.*
```

Comment:
Why wildcard imports?

Reply:
Because IntelliJ adds them and they can't be autofixed

Reply:
It doesn't do that for me but whatever